core/templates/pages/learner-dashboard-page/suggestion-modal/learner-dashboard-suggestion-modal.component.spec.ts

```diff
@@ -24,14 +24,16 @@ import { MockTranslatePipe } from 'tests/unit-test-utils';
 
 import { LearnerDashboardSuggestionModalComponent } from './learner-dashboard-suggestion-modal.component';
 
+// TODO(#9749): Call original AngularHtmlBindWrapperDirective
+// after migration of that directive.
 let MockAngularHtmlBindWrapperDirective = function(
     options: Component): Directive {
   const metadata: Directive = {
     selector: options.selector,
     inputs: options.inputs,
     outputs: options.outputs
   };
-  return <undefined>Directive(metadata)(class _ { });
+  return Directive(metadata)(<never> class _ { });
```

Comment:
Not really sure if this is the right thing to do because directive is used in other spec files by simply injecting